client/src/app/pages/external/jira/tracker-form.tsx

```diff
@@ -57,49 +57,70 @@ interface FormValues {
 }
 
 export interface TrackerFormProps {
-  tracker?: Tracker;
   onClose: () => void;
+  setUpdatingTrackerId: React.Dispatch<
+    React.SetStateAction<number | null | string>
+  >;
+  tracker?: Tracker;
 }
 
 export const TrackerForm: React.FC<TrackerFormProps> = ({
   tracker,
   onClose,
+  setUpdatingTrackerId,
 }) => {
   const { t } = useTranslation();
 
   const [axiosError, setAxiosError] = useState<AxiosError>();
-  const [isLoading, setIsLoading] = useState(false);
 
   const { trackers: trackers } = useFetchTrackers();
   const { identities } = useFetchIdentities();
 
   const { pushNotification } = React.useContext(NotificationsContext);
 
-  const onCreateTrackerSuccess = (_: AxiosResponse<Tracker>) =>
+  const onCreateTrackerSuccess = (_: AxiosResponse<Tracker>) => {
     pushNotification({
       title: t("toastr.success.save", {
         type: t("terms.instance"),
       }),
       variant: "success",
     });
 
-  const onCreateUpdatetrackerError = (error: AxiosError) => {
-    setAxiosError(error);
-  };
+    setUpdatingTrackerId(_.data.id);
 
-  const { mutate: createTracker } = useCreateTrackerMutation(
-    onCreateTrackerSuccess,
-    onCreateUpdatetrackerError
-  );
+    setTimeout(() => {
+      setUpdatingTrackerId(null);
+    }, 5000);
+  };
 
-  const onUpdateTrackerSuccess = (_: AxiosResponse<Tracker>) =>
+  const onUpdateTrackerSuccess = (
+    _: AxiosResponse<Tracker>,
+    tracker: Tracker
+  ) => {
     pushNotification({
       title: t("toastr.success.save", {
         type: t("terms.instance"),
       }),
       variant: "success",
     });
 
+    setUpdatingTrackerId(tracker.id);
+
+    setTimeout(() => {
+      setUpdatingTrackerId(null);
+    }, 5000);
+  };
+
+  const onCreateUpdatetrackerError = (error: AxiosError) => {
+    setAxiosError(error);
+    setUpdatingTrackerId(null);
```

Comment:
I'm not sure this is needed.  If there is an error, the success callback won't set the tracker id.